cluster/local/integration_tests.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+set -e
+
+scriptdir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+projectdir="${scriptdir}/../.."
+
+# get the build environment variables from the special build.vars target in the main makefile
+eval $(make --no-print-directory -C ${projectdir} build.vars)
+BUILD_IMAGE="${BUILD_REGISTRY}/${PROJECT_NAME}-amd64"
+
+helm_tag="$(cat ${projectdir}/_output/version)"
+CROSSPLANE_IMAGE="${DOCKER_REGISTRY}/${PROJECT_NAME}:${helm_tag}"
+
+DEFAULT_NAMESPACE="crossplane-system"
+
+# cleanup on exit
+function cleanup {
+    kind delete cluster
+}
+trap cleanup EXIT
+
+# create cluster
+kind create cluster
+export KUBECONFIG="$(kind get kubeconfig-path --name="kind")"
+
+# tag crossplane image and load it to kind
+docker tag ${BUILD_IMAGE} ${CROSSPLANE_IMAGE}
+kind load docker-image ${CROSSPLANE_IMAGE}
+
+# install helm and tiller
+kubectl apply -f ${scriptdir}/helm-rbac.yaml
+helm init --service-account tiller
+kubectl -n kube-system rollout status deploy/tiller-deploy
+
+# install crossplane
+echo "installing helm package(s) into \"$ns\" namespace"
+helm install --name ${PROJECT_NAME} --namespace ${DEFAULT_NAMESPACE} ${projectdir}/cluster/charts/${PROJECT_NAME} --set image.pullPolicy=Never,imagePullSecrets=''
+
+# wait for creation time
+echo "wait for 5 seconds so that the pods are up and running"
```

Comment:
have you considered using `kubectl rollout` here as well?

Reply:
I would second `kubectl rollout`. Another option could be a polling function to see if / when the pods are either running or encountered an error.